lib/BackbeatConsumer.js

```diff
@@ -180,6 +180,8 @@ class BackbeatConsumer extends EventEmitter {
             // automatically create topic
             'allow.auto.create.topics': true,
             'rebalance_cb': true,
+            // decrease max metadata age to 5s to avoid lag
```

Comment:
"lag" does not really help in this context : you need to explain what _lag_ it avoids...